wakuv2/nwaku.go

```diff
@@ -1895,10 +1912,9 @@ func (w *Waku) ClearEnvelopesCache() {
 	w.envelopeCache = newTTLCache()
 }
 
-// TODO-nwaku
 func (w *Waku) PeerCount() int {
-	return 0
-	// return w.node.PeerCount()
+	peerCount, _ := w.GetNumConnectedPeers()
```

Comment:
I think in this case we shouldn't swallow the error, but instead modify the Signature of `func(w *Waku) PeerCount()` to return `(int, err)` instead